log4j-core/src/main/java/org/apache/logging/log4j/core/config/DefaultConfiguration.java

```diff
@@ -49,4 +61,60 @@ public DefaultConfiguration() {
 
     @Override
     protected void doConfigure() {}
+
+    static Layout<? extends String> createDefaultLayout() {
```

Comment:
Why do we need to publish `DefaultLayout` like this? Can't `DefaultLayout` simply be an individual package-private class in `.core.config` package where `AbstractConfiguration` accesses it via `DefaultLayout.INSTANCE`? I mean, given `DL` is only used by `AC`, I don't see the reason to contain it in `DC`.

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/3118/commits/bf892e169e273d3e9c4e4812e4ffac4178460a3d